public/src/main/java/com/revenuecat/purchases/parceler/JSONObjectParceler.kt

```diff
@@ -5,7 +5,7 @@ import kotlinx.parcelize.Parceler
 import org.json.JSONObject
 
 /** @suppress */
-internal object JSONObjectParceler : Parceler<JSONObject> {
+object JSONObjectParceler : Parceler<JSONObject> {
```

Comment:
As mentioned in the other PR, we probably want to change a few things to not have to make this public if possible. (Mentioning it again so we don't forget)